packages/js/src/generators/typescript-sync/typescript-sync.spec.ts

```diff
@@ -391,6 +391,95 @@ describe('syncGenerator()', () => {
       `);
     });
 
+    it('should not prune existing external project references that are not dependencies but are ignored', async () => {
+      writeJson(tree, 'packages/b/tsconfig.json', {
+        compilerOptions: {
+          composite: true,
+        },
+        references: [
+          { path: './some/thing' },
+          { path: './another/one' },
+          { path: '../../some-path/dir' }, // this is not a dependency but it's ignored, should not be pruned
```

Comment:
Given you need to update this file anyway, please could you make it clear this means git ignored? (I was looking for nx.json updates in the test when I read this)